sormas-e2e-tests/scripts/runTests.sh

```diff
@@ -25,7 +25,7 @@ rm -rf ./allureReports
 echo "Executing gradle clean..."
 ./gradlew clean goJF
 echo "Starting all BDD tests under @$1 tag..."
-./gradlew startTests -Dcucumber.tags=\"@$1\" -Dheadless=true -Dcourgette.threads=9
+./gradlew startTests -Dcucumber.tags=\"@$1\" -Dheadless=true -Dcourgette.threads=9 -DenvConfig=C:/Users/Zack/Desktop/data.json
```

Comment:
envConfig could be parameterized

Reply:
yes, this was commited by mistake to check the regression, we'll hardcode this path into jenkins machine once done